### PR TITLE
Project Structure

### PR DESCRIPTION
**Pull request by vibern0**
This code was written by me a very long time ago. I never worked professionally with python, so I'm not very proficient as I would like to.

In this PR I decided to change the structure, putting everything on the same page and also fixed some naming to a better normalization.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
+[![License][license-svg]][license-url]
+[![Status][status-svg]][status-url]
+
+A simple Point-of-Sale system with graphic interface. The system is controlled by one person at time.
+
+#### Design
+The system is developed in python and designed with [Kivy][kivy]. 
+
+#### How it works
+All the workers are registered on the system, and when someone start to work, needs to login in his account and logout at the end.
```
Comment (JeanCarlosChavarriaHughes) [inline README.md:11]: someone starts

File: README.md [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
+Everything keeps registered.
+The system load from a database what is still available and what's not available. When someting's stock is almost ending, the user will be warned.
```
Comment (JeanCarlosChavarriaHughes) [inline README.md:13]: The system loads ...
When something in the stock is ...

File: README.md [hunk 3]
```diff
@@ -1,29 +1,29 @@
...
+The administrator will take care of the available itens.
```
Comment (JeanCarlosChavarriaHughes) [inline README.md:14]: available items.

File: README.md [hunk 4]
```diff
@@ -1,29 +1,29 @@
...
+
+#### Storing data
+In this project, to save all data, a [SQLite3][sqlite3] local database is used. The database is already included in the project.
+
+#### Manage accounts
+The administrator as a panel to manage accounts in the system. Each employer have a card with a username and a password.
```
Comment (JeanCarlosChavarriaHughes) [inline README.md:20]: Each employee has

**Review by JeanCarlosChavarriaHughes — CHANGES_REQUESTED** [review]

**Comment by vibern0** [comment]
@JeanCarlosChavarriaHughes thanks.
Since your comments were mostly around README file and I was planning to update it, please have a look at #26

**Comment by JeanCarlosChavarriaHughes** [comment]
Nice. #26  and #25 are ready for merge.

**Pull request merged**